docs/MO_DG/prepare_model/convert_model/supported_model_formats.md

```diff
@@ -0,0 +1,37 @@
+# Supported Model Formats {#Supported_Model_Formats}
+
+@sphinxdirective
+
+.. toctree::
+   :maxdepth: 1
+   :hidden:
+
+   openvino_docs_MO_DG_prepare_model_convert_model_Convert_Model_From_TensorFlow
+   openvino_docs_MO_DG_prepare_model_convert_model_Convert_Model_From_ONNX
+   openvino_docs_MO_DG_prepare_model_convert_model_Convert_Model_From_PyTorch
+   openvino_docs_MO_DG_prepare_model_convert_model_Convert_Model_From_Paddle
+   openvino_docs_MO_DG_prepare_model_convert_model_Convert_Model_From_MxNet
+   openvino_docs_MO_DG_prepare_model_convert_model_Convert_Model_From_Caffe
+   openvino_docs_MO_DG_prepare_model_convert_model_Convert_Model_From_Kaldi
+   openvino_docs_MO_DG_prepare_model_convert_model_tutorials
+
+@endsphinxdirective
+
+
+**OpenVINO IR (Intermediate Representation)** - the proprietary format of OpenVINO™, benefiting from the full extent of its features.
+
+**ONNX, PaddlePaddle** - formats supported directly, which means they can be used with OpenVINO Runtime without any prior conversion. For a guide on how to run inference on ONNX and PaddlePaddle, see how to [Integrate OpenVINO™ with Your Application](../../../OV_Runtime_UG/integrate_with_your_application.md).
+
+**TensorFlow, PyTorch, MXNet, Caffe, Kaldi** - formats supported indirectly, which means they need to be converted to one of the formats listed before. Conversion from these formats to OpenVINO IR is performed with Model Optimizer. In some cases other converters need to be used as intermediaries.
```

Comment:
@kblaszczak-intel, is it copied-pasted from somewhere, or it is new content? Overall this block should be reworked completely because it is wrong positioning. We always treat all these formats as directly supported and nobody ever suggested converting, say TensorFlow to PaddlePaddle or ONNX. Model Optimizer is the main way to prepare a model in real high-performance deployment scenarios. From that point, there is no difference between all those formats.

Reply:
This content was based on other articles and used as a node article for several documents on model conversion. If you see any inconsistency or it is not up to date, please suggest which statements need correction.
As far as I've been told, documentation should present the whole picture and using Model Optimizer for IR conversion is not the only workflow possible. If this has changed, please let me know, we need to adjust.
If you confirm that no intermediate conversions are necessary (can we now use pytorch directly?), the last sentence has to be removed, no doubt.